src/sidebar/components/ThreadCard.tsx

```diff
@@ -63,22 +61,30 @@ function ThreadCard({ frameSync, thread }: ThreadCardProps) {
   }, [focusRequest, store, thread.id]);
 
   return (
-    /* eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions */
     <Card
       active={isHovered}
       classes="cursor-pointer focus-visible-ring theme-clean:border-none"
       data-testid="thread-card"
       elementRef={cardRef}
-      tabIndex={-1}
+      tabIndex={0}
+      role="button"
+      aria-label="Press Enter to scroll annotation into viewport"
       onClick={e => {
         // Prevent click events intended for another action from
         // triggering a page scroll.
-        if (!isFromButtonOrLink(e.target as Element) && thread.annotation) {
+        if (!isFromButtonOrLink(e.target as Element)) {
```

Comment:
Are you certain that the `thread.annotation` check here is not necessary? `Thread`s can and do render without annotations. `ThreadCard`s may not, but that also may need to change with the shared-annotations project coming up (we'll need a way to show "headless threads" in a thread card). Anyway, can you help me understand this change?

Reply:
I moved the condition to the `scrollToAnnotation` callback, that is now called from two places, so that we did not have to do that check in two places.

See https://github.com/hypothesis/client/pull/5539#discussion_r1227795482

Maybe it's too subtle, I could do a proper `if (ann) { ... }` instead.

Reply:
Yes, I think it might be helpful to "surface" that check here instead of tucking it away in the abstraction, even though it's not DRY. I think it helps to be able to scan `Thread`/`ThreadCard` to see logic that varies based on whether the thread has an annotation or not.